src/gradata/hooks/_base.py

```
@@ -0,0 +1,93 @@
+        min_profile = meta.get("profile", Profile.STANDARD)
+        if not should_run(min_profile):
+            return
+        raw = raw_input if raw_input is not None else sys.stdin.read()
+        data = read_input(raw)
+        if data is None and meta.get("event") not in ("SessionStart", "Stop", "PreCompact"):
+            return
+        result = main_fn(data or {})
+        if result:
+            print(json.dumps(result))
+    except Exception:
+        pass  # Silent — never break Claude Code
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)